### ISSUE TITLE
[Table] md-numeric dosn't work if cell contains Vue component

### ISSUE THREAD

**Issue by flesser** [vuematerial/vue-material#570]
According to the docs, `md-numeric` should align table cell contents to the right.

However, having a Vue component inside the cell (resulting in `this.$children.length > 0` in [mdTableCell.vue:27](https://github.com/marcosmoura/vue-material/blob/master/src/components/mdTable/mdTableCell.vue)) adds the CSS class `md-has-action`, wich sets `justify-content: space-between;` ([mdTable.scss:163](https://github.com/marcosmoura/vue-material/blob/master/src/components/mdTable/mdTable.scss#L163)), thus overriding md-numeric's `justify-content: flex-end;`

### Steps to reproduce
Put an Icon next to the cell content:
```html
<md-table-cell md-numeric>
   10
   <md-icon>home</md-icon>
</md-table-cell>
```

Full example:
https://codepen.io/anon/pen/xqGPvW

### Which browser?
Vue Material 0.7.1, Firefox 51.0.1, Linux

### What is expected?
Cell content is aligned right

### What is actually happening?
Cell content is aligned left

**Comment by pablohpsilva** [comment]
Hello @flesser ! How are you?
Dude, thank you for letting us know about this issue. We'll look into it and fix it ASAP, OK?

**Comment by pablohpsilva** [comment]
Probably fixed on #787
